docs/tutorials/shadow-traffic.md

```diff
@@ -0,0 +1,65 @@
+# Shadow Traffic
+
+This tutorial will show how to setup routing for shadow traffic, where one backend (main) will receive the full
+traffic, while a shadowing backend (test) will receive only a certain percentage of the same traffic.
+
+#### Used Predicates:
+
+* [Tee](../../reference/predicates/#tee)
```

Comment:
IIRC we should use `../../reference/predicates.md#tee`

Reply:
please also for the other links :)

Reply:
will check :+1:

Reply:
indeed. Both work, but if the links are in the suggested way, then they are also validated. Thx!